src/pudl/metadata/resources/ferc1.py

```diff
@@ -1857,24 +1883,28 @@
         "field_namespace": "ferc1",
     },
     "out_ferc1__yearly_detailed_income_statements": {
-        "description": (
-            "This table contains granular accounting data from utilities' income statements."
-            "This table is derived from four FERC Form 1 accounting tables with nested calculations:\n\n"
-            " * `core_ferc1__yearly_income_statements_sched114`\n"
-            " * `core_ferc1__yearly_depreciation_summary_sched336`\n"
-            " * `core_ferc1__yearly_operating_expenses_sched320`\n"
-            " * `core_ferc1__yearly_operating_revenues_sched300`\n\n"
-            "We reconciled the nested calculations within these tables and then identified the "
-            "most granular data across the tables.\n"
-            "We applied slight modifications to two columns (utility_type & plant_function)"
-            "as compared to the originally reported values in our core tables. "
-            "The modifications were applied to either provide more specificity (i.e. we converted"
-            "some `total` utility_type's into `electric`) or to condense similar categories "
-            "for easier analysis (i.e. creating a `hydraulic_production` plant_function by "
-            "combining `hydraulic_production_conventional` and `hydraulic_production_pumped_storage`\n"
-            "See ``pudl.output.ferc1.Exploder`` for more details. This table was made entirely with "
-            "support and direction from RMI."
-        ),
+        "description": {
+            "additional_summary_text": "granular, de-duplicated accounting data of utilities' income statements.",
+            "additional_source_text": "(Schedule 114, 300, 320 and 336)",
```

Comment:
Sneaky missing plural
```suggestion
            "additional_source_text": "(Schedules 114, 300, 320 and 336)",
```